### PR TITLE
Bug 1221899 - configure device-asus-fugu

### PR DESCRIPTION
**Pull request by fatseng**

### CODE REVIEW

File: devices.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "id": "Nexusplayer",
+    "adb": {
+      "ro.product.device": [ "fugu" ],
+      "ro.bootloader": ["FUGU-02.11"],
+      "ro.build.id": ["MMB29T", "LMY48M", "LMY47D"]
```
Comment (lissyx) [inline devices.json]: Are you sure about that?

File: full_fugu.mk [hunk 2]
```diff
@@ -14,10 +14,24 @@
 # limitations under the License.
 #
 $(call inherit-product, device/asus/fugu/device.mk)
-$(call inherit-product, device/google/atv/products/atv_base.mk)
+#$(call inherit-product, device/google/atv/products/atv_base.mk)
```
Comment (lissyx) [inline full_fugu.mk]: Why do you need this ?

File: device.mk [hunk 3]
```diff
@@ -244,7 +244,14 @@ PRODUCT_COPY_FILES += \
     device/asus/fugu/init.fugu.diag.rc.userdebug:root/init.fugu.diag.rc
 endif
 
-$(call inherit-product-if-exists, vendor/asus/fugu/device-vendor.mk)
+#$(call inherit-product-if-exists, vendor/asus/fugu/device-vendor.mk)
```
Comment (lissyx) [inline device.mk:247]: Why do you need this also?

File: devices.json (hunk 1 above)
Comment (fatseng) [inline devices.json]: I get them from nexus player which is 6.0.1 version. [ro.product.device]: [fugu], [ro.bootloader]: [FUGU-02.11], [ro.build.id]: [MMB29T]

File: full_fugu.mk (hunk 2 above)
Comment (fatseng) [inline full_fugu.mk]: It causes build failed since it tries to build libfilterpack_imageproc.

File: device.mk (hunk 3 above)
Comment (fatseng) [inline device.mk:247]: device-vendor.mk installs AtvRemoteService.apk  LeanbackLauncher.apk  TV.apk. We don't need it.

File: devices.json (hunk 1 above)
Comment (lissyx) [inline devices.json]: Yes, but it's not enough. Are you sure a build done with your tree will work with those L and M releases ?

I'm not sure that blobs will be compatible between M and L for example

File: full_fugu.mk (hunk 2 above)
Comment (lissyx) [inline full_fugu.mk]: Can't we just make sure PRODUCT_PACKAGES is controlled? I fear we might miss other bits.

**Pull request merged**